packages/zeplin-extension-style-kit/values/gradient/radialGradient.js

```diff
@@ -0,0 +1,58 @@
+import LinearColorStop from "./linearColorStop";
+import Percent from "../percent";
+
+class RadialGradient {
+    // eslint-disable-next-line max-params
+    constructor(colorStops, from, to, width, height) {
+        this.center = from;
+
+        const x1 = from.x * width;
+        const y1 = from.y * height;
+        const x2 = to.x * width;
+        const y2 = to.y * height;
+
+        const gradientDistance = Math.hypot(x1 - x2, y1 - y2);
```

Comment:
It could be nice to describe the logic behind calculating gradients in comments.